### PR TITLE
Increase the max threads from 32 to 256

### PR DESCRIPTION
**Pull request by chraac**
In #5292, we've requested to increase the `threads_batch` in ui to allow `> 32` threads, and here's the PR, to enlarge it.

Minor changes, tested on my rig with `llama.cpp` loader, work as expected.
947559cc1db69b66a2950aefcb6f8a9

## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/oobabooga/text-generation-webui/wiki/Contributing-guidelines).

### CODE REVIEW

File: modules/ui_model_menu.py [hunk 1]
```diff
@@ -94,8 +94,8 @@ def create_ui():
                             shared.gradio['n_ctx'] = gr.Slider(minimum=0, maximum=shared.settings['truncation_length_max'], step=256, label="n_ctx", value=shared.args.n_ctx, info='Context length. Try lowering this if you run out of memory while loading the model.')
                             shared.gradio['tensor_split'] = gr.Textbox(label='tensor_split', info='List of proportions to split the model across multiple GPUs. Example: 18,17')
                             shared.gradio['n_batch'] = gr.Slider(label="n_batch", minimum=1, maximum=2048, step=1, value=shared.args.n_batch)
-                            shared.gradio['threads'] = gr.Slider(label="threads", minimum=0, step=1, maximum=32, value=shared.args.threads)
-                            shared.gradio['threads_batch'] = gr.Slider(label="threads_batch", minimum=0, step=1, maximum=32, value=shared.args.threads_batch)
+                            shared.gradio['threads'] = gr.Slider(label="threads", minimum=0, step=1, maximum=256, value=shared.args.threads)
+                            shared.gradio['threads_batch'] = gr.Slider(label="threads_batch", minimum=0, step=1, maximum=256, value=shared.args.threads_batch)
```
Comment (chraac) [inline modules/ui_model_menu.py:98]: Further thought, should we set the `maximum` base on the cpu cores? Like `maximum=multiprocessing.cpu_count()`

**Comment by Ph0rk0z** [comment]
I think sometimes using more threads than physical cores is desirable. For some intel CPUs. Depends on your setup.

**Comment by chraac** [comment]
> I think sometimes using more threads than physical cores is desirable. For some intel CPUs. Depends on your setup.

hmm, then I'll leave the `maximum` to `256`, let the user decide how many threads they want to use

**Comment by chraac** [comment]
@Ph0rk0z coulde you please merge this PR regarding its a trival change, verified on my machine, and also the linter show no errors.

**Comment by Ph0rk0z** [comment]
Heh, I can't merge. Its on oobabooga.

**Comment by chraac** [comment]
> Heh, I can't merge. Its on oobabooga. 

Ahh, got it, anyway, thanks for the comments

**Comment by end-me-please** [comment]
I'd also like to see this merged

**Comment by chraac** [comment]
@oobabooga , sorry for bothering you, could you please have a look on this PR when you have a monent? I'm open to any feedback you might have and am ready to make any necessary adjustments.. Thanks!

**Comment by oobabooga** [comment]
Thanks for the PR -- it's a fair change, I just didn't have the time to review it before. Merging now.

**Pull request merged**